AmethystAPI/src/amethyst/Utility.cpp

```diff
@@ -1,4 +1,11 @@
 #include "amethyst/Utility.h"
+#include <codecvt>
+#include <locale>
+
+// std::wstring_convert<std::codecvt_utf8<wchar_t>> is deprecated
+// but there is no replacement for it currently (as of C++23)
+// TODO find a working replacement
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
```

Comment:
Look into what Intop suggested. It definitely is a thing you can do though, as I've done it in the past. This should be removed.